tsconfig-build.json

```diff
@@ -28,6 +28,7 @@
         {"path": "./packages/wonder-blocks-progress-spinner/tsconfig-build.json"},
         {"path": "./packages/wonder-blocks-search-field/tsconfig-build.json"},
         {"path": "./packages/wonder-blocks-switch/tsconfig-build.json"},
+        {"path": "./packages/wonder-blocks-tabs/tsconfig-build.json"},
```

Comment:
Needed this so we generate types for the `wonder-blocks-tabs` package!

cc: @marcysutton you might need to update this too for the announcer package!